### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by astridu** [sul-dlss/stanford-arclight#651]
Here are the issues that site improve picked up by page. 

GLOBAL
1. "Stanford University" header - text not included in an ARIA landmark
All perceivable text content should be included in an [ARIA landmark](https://www.w3.org/WAI/ARIA/apg/practices/landmark-regions/). Perceivable text content refers to text that can be perceived by users of assistive technology — some of which may be invisible to sighted users.

HOMEPAGE
1. Page needs to start with level 1 heading. "Find archival materials"
2. Change the italicized text under the featured image to regular paragraph text. Apparently italicized characters can be difficult for people with dyslexia to read, and it's not really needed here.

BROWSE ALL COLLECTIONS
1. a couple ARIA attributes unsupported or not allowed:
a) aria-label on span (on the paginator)
b) aria-labelledby on div (in an open facet in left nav)

BROWSE ALL REPOSITORIES
1. It's complaining that we skipped an H2 heading (There's an H1 and H3). I propose we make the headings in the cards H2 headings, and then change the H2 heading "About this site" on the home page to match the card heading style.

INDIVIDUAL REPOSITORY HOME
1. a couple ARIA attributes unsupported or not allowed:
a) aria-label on span (on the paginator)
b) aria-labelledby on div (in an open facet in left nav)

**Comment by taylor-steve** [comment]
Adding some additional issues I'll try to address:

- Any page with facets like `/collections` at mobile size: Visible label and accessible name do not match "show facets" vs "toggle facets"
- When "Collection contents" is empty on a collection, there's an aria error

**Comment by taylor-steve** [comment]
The aria-label/aria-labelledby facet and paginator issues are fixed upstream in Blacklight, but not yet in a v8 release:

https://github.com/projectblacklight/blacklight/commit/4620e0b77581a172702b32157c7c538897984c43
https://github.com/projectblacklight/blacklight/commit/d1e7e12c09224f4c48e20822effb9b09c058f42b
